cargo-dylint/tests/ci.rs

```diff
@@ -133,6 +133,165 @@ fn cargo_dylint_and_dylint_readmes_are_equal() {
     compare_lines(&cargo_dylint_readme, &dylint_readme);
 }
 
+#[test]
+fn examples_readme_contents() {
+    let examples_dir = find_examples_dir();
+    let categories = vec![
+        "general",
+        "supplementary",
+        "restriction",
+        "experimental",
+        "testing",
+    ];
+    
+    // Read the existing README
+    let readme_path = examples_dir.join("README.md");
+    let readme_content = read_to_string(&readme_path).unwrap();
+
+    // Generate just the lint description tables
+    let tables = generate_lint_tables(&examples_dir, &categories);
+    
+    // Extract the current tables section from README using markers
+    let current_tables = extract_between_markers(&readme_content, 
```

Comment:
```suggestion
    let actual_tables = extract_between_markers(&readme_content, 
```
Nit to follow the convention that "expected" is often paired with "actual".